Gemfile.template

```diff
@@ -45,5 +44,99 @@ gem "thwait"
 gem "bigdecimal", "~> 3.1"
 
 # Pin plugins to avoid including those with breaking changes in development builds without a defined lockfile
-gem "logstash-input-beats", "~> 6"
-gem "logstash-output-http", "~> 5"
+gem "logstash-codec-avro", "~> 3.4", :require => false
```

Comment:
When do we update these plugin versions?

Reply:
The optimistic `~>` pin means we don't need to. This keeps the `8.x` branch aligned with the major versions and at least the minor that we are shipping in 8.17.0, and should not be included in any released artifacts.

Reply:
Ah target branch is `8.x` 🤦!